cmd/gpu_plugin/gpu_plugin.go

```diff
@@ -79,6 +83,17 @@ func newDevicePlugin(sysfsDir, devfsDir string, options cliOptions) *devicePlugi
 		scanTicker:       time.NewTicker(scanPeriod),
 		scanDone:         make(chan bool, 1), // buffered as we may send to it before Scan starts receiving from it
 	}
+
+	if options.resourceManagement {
```

Comment:
This function is not as trivial as it used to be. Would be nice to test it too.

Reply:
ok

Reply:
This is a can of worms. There's no clean way except a simple separate test, other options mean a lot of refactoring. I added a separate test.